draft-ietf-quic-recovery.md

```diff
@@ -532,7 +532,7 @@ delay sending an acknowledgement.
 
 When the PTO is armed for Initial or Handshake packet number spaces, the
 max_ack_delay in the PTO period computation is set to 0, since the peer is
-expected to acknowledge these packets immediately; see 13.2.1 of
+expected to not delay these packets intentionally; see 13.2.1 of
```

Comment:
I think the old text is better, I'll need to re-read the entire text to understand my comment.

```suggestion
expected to acknowledge these packets immediately; see 13.2.1 of
```

Reply:
After a re-read, I think I was thinking this was a problem for packets in ApplicationData, but after re-thinking this, I don't think there's an issue there.  I posted this 1 minute before our editors meeting, so I suspect I was a bit rushed.